package.json

```diff
@@ -15,7 +15,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test --transformIgnorePatterns",
     "eject": "react-scripts eject",
-    "test:coverage": "rimraf coverage && react-scripts test --coverage --watchAll=false && npm run cypress:run && node ./scripts/coverage-merge.js && nyc report",
+    "test:coverage": "rimraf coverage && npm test -- --coverage --watchAll=false && npm run cypress && npm run test:merge-coverage && nyc report",
```

Comment:
it's just a small suggestion, you can take a look on [concurrently](https://www.npmjs.com/package/concurrently) package to help the script shorter

Besides, I don't think we have to remove the `coverage` folder at the beginning

Reply:
@nvminhtue nice, I will have a try on `concurrently` package 👍 Thanks for the suggestion 😄 
`rimraf coverage` was in the template, let me remove it